babel.config.js

```diff
@@ -88,6 +90,14 @@ module.exports = api => {
         {
           modules,
           targets,
+          loose: !isTest,
```

Comment:
Why do we use a different `loose` value in test environment? That seems error prone.